File: WebAppVen/Models/WB/WB_MM_PO_M.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebAppVen.Models
{
    public class WB_MM_PO_M : JCLib.Mvc.BaseModel
    {
        public string PO_NO { get; set; }
        public string AGEN_NO { get; set; }
        public string PO_TIME { get; set; }
        public string M_CONTID { get; set; }
        public string PO_STATUS { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string MEMO { get; set; }
        public string ISCONFIRM { get; set; }
        public string ISBACK { get; set; }
        public string PHONE { get; set; }
        public string SMEMO { get; set; }
        public string ISCOPY { get; set; }

        public string MMCODE { get; set; }
        public string PO_QTY { get; set; }
        public string PO_PRICE { get; set; }
        public string M_PURUN { get; set; }
        public string M_AGENLAB { get; set; }
        public string PO_AMT { get; set; }
        public string M_DISCPERC { get; set; }
        public string UNIT_SWAP { get; set; }
        public string MMNAME_C { get; set; }
        public string MMNAME_E { get; set; }


        public string ISCR { get; set; }
    }
}

File: WebAppVen/Scripts/app/model/WB_MM_PO_M.js
Ext.define('WEBAPP.model.WB_MM_PO_M', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'PO_NO', type: 'string' },
        { name: 'AGEN_NO', type: 'string' },
        { name: 'PO_TIME', type: 'string' },
        { name: 'M_CONTID', type: 'string' },
        { name: 'PO_STATUS', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },
        { name: 'MEMO', type: 'string' },
        { name: 'ISCONFIRM', type: 'string' },
        { name: 'ISBACK', type: 'string' },
        { name: 'PHONE', type: 'string' },
        { name: 'SMEMO', type: 'string' },
        { name: 'ISCOPY', type: 'string' },

        { name: 'MMCODE', type: 'string' },
        { name: 'PO_QTY', type: 'string' },
        { name: 'PO_PRICE', type: 'string' },
        { name: 'M_PURUN', type: 'string' },
        { name: 'M_AGENLAB', type: 'string' },
        { name: 'PO_AMT', type: 'string' },
        { name: 'M_DISCPERC', type: 'string' },
        { name: 'UNIT_SWAP', type: 'string' },
        { name: 'MMNAME_C', type: 'string' },
        { name: 'MMNAME_E', type: 'string' }
    ]
});